src/main.cc

```diff
@@ -494,6 +494,8 @@ void register_options()
                        WriteMethod::Overwrite);
     reg.declare_option<int, check_timeout>(
         "idle_timeout", "timeout, in milliseconds, before idle hooks are triggered", 50);
+    reg.declare_option<int, check_timeout>(
+        "next_key_idle_timeout", "timeout, in milliseconds, before next-key suggestions are displayed", 250);
```

Comment:
I am not sure it makes sense to make the next_key timeout special, we have auto-info displaying for prompt commands on idle timeout as well, I am not sure why we would have a separate timeout for next-key.

Reply:
Done